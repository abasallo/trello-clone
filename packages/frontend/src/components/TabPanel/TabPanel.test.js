import React from 'react'
import renderer from 'react-test-renderer'

import TabPanel from './TabPanel'

test('Renders properly', () => expect(renderer.create(<TabPanel index={0} value={0} />).toJSON()).toMatchSnapshot())
