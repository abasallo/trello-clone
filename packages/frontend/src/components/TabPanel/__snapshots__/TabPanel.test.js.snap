// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders properly 1`] = `
<div
  aria-labelledby="simple-tab-0"
  hidden={false}
  id="simple-tabpanel-0"
  role="tabpanel"
>
  <div
    className="MuiBox-root MuiBox-root-1"
  >
    <p
      className="MuiTypography-root MuiTypography-body1"
    />
  </div>
</div>
`;
